docs/examples/middleware/builtin/csrf_httpx.py

```diff
@@ -0,0 +1,21 @@
+import httpx
+
+with httpx.Client() as client:
+    get_response = client.get("http://localhost:8000/")
+
+    # "csrftoken" is the default cookie name
+    csrf = get_response.cookies["csrftoken"]
+
+    # "x-csrftoken" is the default header name
+    post_response_using_header = client.post("http://localhost:8000/", headers={"x-csrftoken": csrf})
```

Comment:
this is a mistake I made when I wrote this, its not on you, but can you change this ? 🙂 
```suggestion
    post_response_using_header = client.post("http://localhost:8000/1", headers={"x-csrftoken": csrf})
```

Reply:
Done